cronjob.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+CURL='/usr/bin/curl'
+CRON_HTTP="https://{$AUTH_USERNAME}:{$AUTH_PASSWORD}@{$CONNECT_DOMAIN}/cron/{$CRON_HASH}"
```

Comment:
Best add a conditional here to make sure the AUTH vars are there before using them. That way, if they are set per environment they get used or not.

Reply:
Added in 66b951b